### ISSUE TITLE
[TalkBack] When you set a PIN does not read the PIN.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#231]
```
What steps will reproduce the problem?

Procedure:
[Settings]
1 If set to Screen lock PIN.
Test Order
1 Enters the Power / Lock key 2 times a particular App screen, press the Lock 
screen screen.
2 Release the lock by entering the correct PIN number. 

When you set a PIN does not read the PIN.

What is the expected output? What do you see instead?
When you set a PIN read the PIN.


What version of the product are you using? On what operating system?

4.2.1

Please provide any additional information below.

```

Original issue reported on code.google.com by `ezc...@gmail.com` on 19 Apr 2013 at 6:00

**Comment by GoogleCodeExporter** [comment]
```
Not sure I'm understanding this report correctly.

TalkBack will not speak passwords unless you connect a headset or select the 
"Speak Passwords" option in Settings > Accessibility.
```

Original comment by `caseybur...@google.com` on 21 Apr 2013 at 9:29
- Changed state: **Invalid**
